pyproject.toml

```diff
@@ -35,7 +43,7 @@ typeCheckingMode = "basic"
 pythonVersion = "3.12"
 include = ["**/*.py"]
 exclude = ["**/node_modules", "**/__pycache__"]
-ignore = ["syncdata.py", "src/oldstuff"]  # Changed to ignore the specific file with dirsync import
+ignore = ["syncdata.py", "src/oldstuff", "nhs/data/join_on_sa1.py"]
```

Comment:
why is this being added?

Reply:
![image](https://github.com/user-attachments/assets/267c9430-55fe-49a4-a0d6-7e63367ae134)


Pylance goes ballistic, saying things like GeoDataFrame isn't a known attribute of geopandas, that geopandas can't read files, type `isnull() is unknown` etc.

Reply:
I think I could fix all of them but it'll take ages